src/provisioner/provisioner.py

```diff
@@ -56,11 +60,13 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None):
 
     @staticmethod
     def config_apply_cortx(cortx_config_store):
+        Log.info('Applying cortx config')
         # TODO: To convert CORTX config
         cortx_config_store.set('cortx', Conf.get(CortxProvisioner._solution_index, 'cortx'))
 
     @staticmethod
     def config_apply_cluster(cortx_config_store):
+        Log.info('Applying cluster config')
```

Comment:
Same here. If we cant print meaningful messages then convert to debug mode. 

Also, there should be an option to turn on the debug mode. Default is info. 
methods
1. Env Variable CORTX_PROVISIONER_DEBUG_LEVEL=info|debug
2. Command line option

Reply:
cortx-setup initialize logger. So each time cortx-setup command executed, we need to add the log_level. Otherwise default(INFO) will be proceeded next time. So command line option is not feasible. 

I'm considering method 1 and the env variable CORTX_PROVISIONER_DEBUG_LEVEL will be set during RPM build. User can set log level from here. So it will be one time action over cortx-setup commands.

Reply:
Done.